Hack-A-Sat 2021/Rapid Unplanned Disassembly/Grade F Prime Beef/grade_f_write_up.md

```diff
@@ -149,7 +149,7 @@ PWD=/home/space/fprimeLOGNAME=spaceHOME=/home/spaceFLAG=flag{sierra457871juliet2
 _XXXXXXXXXXXXXXXX_XXXXXXXXXXXXXXXXXXXXXXXX_XXXXXXXXXXXXXXXX}
 USER=spaceSHLVL=0PATH=/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/gamesMAIL=/var/mail/spaceOLDPWD=/home/space_=./sa
 tellite.exe
-```
+``` 
```

Comment:
Did you mean to put the following couple commands (lines 157-158) in a code block?

Reply:
Markdown on my IDE act funny - committed the changes.